backend/src/v1/staging_complaint/staging_complaint.controller.ts

```diff
@@ -14,15 +15,31 @@ import { Public } from "../../auth/decorators/public.decorator";
 export class StagingComplaintController {
   constructor(private readonly stagingComplaintService: StagingComplaintService) {}
 
-  @Post()
+  @Post("/creates")
```

Comment:
is this a typo? ```@Post("/creates")``` vs ```@Post("/create")```
similar question about ```@Post("/updates")``` vs ```@Post("/update")```

Reply:
I like this question.  I agree that we need something better, but I don't know what.  The thing here is that these routes are used to Post a create record and to Post an update record.  Both are posts (not a patch for update, because it's actually creating new data).  More specifically, we're Posting either a WebEOC create or Posting a WebEOC update.  I don't like using /create or /update because those are verbs.  Maybe webeoc-complaint and webeoc-complaint-update?

Reply:
Might have our lines crossed here. My question wasn't about the the verb you're using, in fact I think POST is 100% correct in either case honestly, but rather the pluralization of your action name. There's something that feels weird about "creates" over just "create"

Reply:
I agree, I don't like creates.  I also don't like create though, since it's a verb.  I'm thinking webeoc-complaint, and webeoc-complaint-update works better since we're posting either a webeoc-complaint or a webeoc-complaint-update.  I've made that change, let me know what ya think!